plugins/outputs/application_insights/application_insights.go

```diff
@@ -283,11 +283,9 @@ func toFloat64(value interface{}) (float64, error) {
 func toInt(value interface{}) (int, error) {
 	if !is32BitChecked {
 		is32BitChecked = true
-		var i int
-		if unsafe.Sizeof(i) == 4 {
+		is32Bit = false
+		if strconv.IntSize == 32 {
 			is32Bit = true
-		} else {
-			is32Bit = false
 		}
 	}
```

Comment:
Does the code block after this section even makes sense?

Reply:
@Hipska 
I think it does.
Function wants to have value of type `int64` or `uint64` as an argument and needs to return `int` (which is by default signed and with size of `32` or `64` bits - depending on CPU/OS environment).

Function will return error if 64-bit `value` is out of range for `int` (for current environment) or it is not `int64` (nor `uint64`).

Look, here function returns error on 64-bit machine and tests passes:
```
func Test_toInt(t *testing.T) {
	var value uint64
	value = math.MaxInt64 + 1

	_, err := toInt(value)
	require.Error(t, err)
}

```

Reply:
> On the other hand, Isn't there any function that does this in the standard libraries already?

I have no idea :)